quickstart/GettingStartedPi.html

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Raspberry Pi
+categories: [opendds]
+tags: []
+order: 12
+---
+The goal of this guide is help you download and compile OpenDDS for Linux on the <a hef="https://www.raspberrypi.org/">Raspberry Pi</a> and run a simple example. 
+<div class="bodytext" align="justify">
+  <a name="Build Directions"><span class="headertext">Build Directions</span></a>
+  <ol>
+    <p>
+      For this guide you need a Raspberry Pi set up with Raspbian Linux and connected to a local network.
+      You will also need a Linux host system connected to the same network where we will build OpenDDS.
+      This guide also assumes you know the address and user credentials of the Pi.
+      Perform these all these steps on the host computer:
+    </p>
+    <li><p>Ensure that your host environment has:</p>
+      <ul>
+        <li>a C++ compiler</li>
+        <li>GNU Make</li>
+        <li>Perl</li>
+        <li>Optional Java SDK 1.5 through 1.8 for Java JNI binding support</li>
+      </ul>
+      <p>See <a href="../documents/building.html">Building</a> for a complete list of dependencies and supported platforms.</p>
+    </li>
+    <li>Download and extract the latest tar.gz file from the <a href="http://download.objectcomputing.com/OpenDDS/">download site</a>.</li>
+    <li>
+      Download and extract the <a href="https://github.com/raspberrypi/tools">Raspberry Pi toolchain from GitHub</a>.
+      Copy or note the absolute path of it, because we need to pass that to OpenDDS's build system.
+    </li>
+    <li>Enter the OpenDDS-&lt;version&gt; directory</li>
+    <li>
+      Run the following as a single command, replacing &lt;PITOOLS&gt; with the absolute path of the toolchain:
+      <pre><code>./configure --target=linux-cross\</br>  --target-compiler="&lt;PITOOLS&gt;/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/arm-linux-gnueabihf-g++"</code></pre>
```

Comment:
Since Java is mentioned above, add a note about `--java` on this step.  (see http://opendds.org/quickstart/GettingStartedLinux.html for an example)